build/gen_stub.php

```diff
@@ -2605,11 +2603,6 @@ protected function getVariableTypeName(): string
         return "constant";
     }
 
-    protected function getVariableTypeCode(): string
```

Comment:
VariableLike::getTypeCode() should have called `getVariableTypeCode()` instead of `getVariableTypeName()`, but since the difference is minimal, I'm fine the removal of `getVariableTypeCode()`.